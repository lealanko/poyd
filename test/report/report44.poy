(* Build, transform to static characters, and report memory *)
read ("1.fas")
build (1)
transform ((all, static_approx))
report (memory)
