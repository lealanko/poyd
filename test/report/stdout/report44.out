Memory usage:
  
  Minor Words: 28277528.
  Promoted Words: 7152358.
  Major Words: 10296735.
  Major Collections: 37
  Heap Words: 2150400
  Heap Chunks: 35
  Live Words: 2130694
  Free Words: 18289
  Free Blocks: 798
  Largest Free: 16695
  Fragments: 1417
  Campactions: 2
  Top Heap Words: 2334720